### PR TITLE
did CodeChall01

### PR DESCRIPTION
**Pull request by Arkuris**

### CODE REVIEW

**Comment by Arkuris** [comment]
- [ yes] Top-level README “Table of Contents” is updated
 - [ yes] README for this challenge is complete
       - [yes ] Summary, Description, Approach & Efficiency, Solution
       - [yes] Picture of whiteboard
       - [ yes?] Link to code
 - [ yes] Feature tasks for this challenge are completed
 - [ n/a ] Unit tests written and passing
       - [ n/a ] “Happy Path” - Expected outcome
       - [ n/a ] Expected failure
       - [ n/a ] Edge Case (if applicable/obvious)

File: javascript/array-reverse/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+<!-- Description of the challenge -->
+Write a function called reverseArray which takes an array as an argument. Without utilizing any of the built-in methods available to your language, return an array with elements in reversed order.
+
+## Whiteboard Process
+<!-- Embedded whiteboard image -->
+![whiteboard](../assets/whiteboard%20CodeChallange%2001.jpg)
+
+## Approach & Efficiency
+<!-- What approach did you take? Why? What is the Big O space/time for this approach? -->
+Talked with classmates, ChatGPT, old code, chatGPT, google.
```
Comment (arpatterson31) [inline javascript/array-reverse/README.md:11]: Hey David! Great job with your first whiteboard! This section is going to be important during interviews. This is going to help the interviewer see how you think through and problem solve.
